tests/checkpoints_test.py

```diff
@@ -114,7 +115,7 @@ def test_safe_normpath(self):
       self.assertEqual(expect, checkpoints.safe_normpath(test))
 
   def test_save_restore_checkpoints(self):
-    tmp_dir = self.create_tempdir().full_path
+    tmp_dir = pathlib.Path(self.create_tempdir().full_path)
```

Comment:
It would be nice to have a test here that checks the new behavior.

Reply:
Without the change (added line 213 in `checkpoints.py`) this updated test would have failed.
Note that the regex parsing is already tested in `test_safe_normpath()`.

Reply:
Makes sense!